pkg/odo/cli/storage/list.go

```diff
@@ -54,22 +61,102 @@ func (o *StorageListOptions) Complete(name string, cmd *cobra.Command, args []st
 
 // Validate validates the StorageListOptions based on completed values
 func (o *StorageListOptions) Validate() (err error) {
-	return
+	switch o.outputFlag {
+	case "", "json":
```

Comment:
It would be good to move this into generic function, as it will be used in more places that just here.